recipe/meta.yaml

```
@@ -35,20 +35,28 @@ test:
     - repo_review.fixtures
     - repo_review.schema
   source_files:
+    - docs
     - tests
+    - README.md
+    - LICENSE
+    - noxfile.py
     - pyproject.toml
+    - .github
+    - .pre-commit-config.yaml
+    - .readthedocs.yaml
   requires:
     - rich-click
     - pip
     - pytest
-    # - sp-repo-review
+    - sp-repo-review
     - validate-pyproject >=0.14
   commands:
     - pip check
     - repo-review --help
     - validate-pyproject -E repo-review --help
```

```suggestion
    - validate-pyproject -E repo-review --help
    - mkdir src
```

You could include the src folder, or just drop an empty folder with the same name in, that should work too.